kit/torch/data.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 from abc import abstractmethod
-from typing import Iterator, Sequence, Sized
+from typing import Iterator, Literal, Sequence, Sized
```

Comment:
I want to keep palkit on python 3.7 for now.

```suggestion
from typing import Iterator, Sequence, Sized
from typing_extensions import Literal
```